crates/sui-core/src/authority.rs

```diff
@@ -1826,6 +1833,49 @@ impl AuthorityState {
         self.epoch_store_for_testing().epoch()
     }
 
+    pub fn checkpoint_all_dbs(
+        &self,
+        checkpoint_path: &Path,
+        cur_epoch_store: &AuthorityPerEpochStore,
+    ) -> SuiResult {
+        let _metrics_guard = self.metrics.db_checkpoint_latency.start_timer();
+        let current_epoch = self.db().perpetual_tables.get_recovery_epoch_at_restart()?;
```

Comment:
cur_epoch_store will give you the current epoch

Reply:
done